domains/nabin.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "nabinkatwal7",
+        "email":"mediocampistaa@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://portfolio-v3-six-iota.vercel.app/",
```

```suggestion
        "CNAME": "portfolio-v3-six-iota.vercel.app"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).